tests/data/output/autogenerated_rsmeval_config.json

```diff
@@ -0,0 +1,34 @@
+{
+    // REQUIRED: replace "ENTER_VALUE_HERE" with the appropriate value!
```

Comment:
Shall we add a comment with `//[link to documentation]` ?

Reply:
Done.